src/flake8_simplify/plugins/ast_if.rs

```diff
@@ -1,8 +1,18 @@
 use rustpython_ast::{Constant, Expr, ExprKind, Stmt, StmtKind};
 
 use crate::ast::types::Range;
+use crate::autofix::Fix;
 use crate::checkers::ast::Checker;
-use crate::registry::{Check, CheckKind};
+use crate::registry::{Check, CheckCode, CheckKind};
+use crate::source_code_generator::SourceCodeGenerator;
+use crate::source_code_style::SourceCodeStyleDetector;
+
+/// Generate source code from an `Expr`.
+fn to_source(expr: &Expr, stylist: &SourceCodeStyleDetector) -> String {
```

Comment:
Duplicate of https://github.com/charliermarsh/ruff/blob/cecd4b166cbb17e784ffb22c3bd966743fb30b3d/src/flake8_simplify/plugins/ast_bool_op.rs#L25-L30, maybe we should move it to ast helpers?